### DISCUSSION TITLE
pnpm as Node.js package manager in Functions

### DISCUSSION THREAD

**Discussion by typed-sigterm** [appwrite/appwrite#8763] [Q&A] [answered]
Hello 👋 Thanks Appwrite team, it looks amazing! I'm considering using Appwrite in my next project.
According to the [docs](https://appwrite.io/docs/products/functions/develop#dependencies), It seems that Appwrite doesn't support pnpm as Node.js package manager. Is there a way to make it? Due to some incompatibility issues between them, if I use pnpm in the project, the dependencies installed on the production may differ from what is expected for pnpm.

**Comment by eldadfux** [comment]
I'll share this with the team, and see what we can do.

**Comment by silicia-apps** [comment, marked as answer]
you can put:

npx pnpm install && npx pnpm build

in build settings - commands of node runtime
